src/test/java/com/productdock/library/gateway/service/BookServiceShould.java

```diff
@@ -28,13 +29,16 @@ class BookServiceShould {
     private static final String BOOK_ID = "1";
     private static final String BOOK_TITLE = "::title::";
     private static final String BOOK_AUTHOR = "::author::";
-    private static final String JWT_TOKEN = "";
+    private static final String JWT_TOKEN = "eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJlbWFpbCI6ImVtYWlsIn0.9_82fPfiHdHoOkyx8WQY8FsgPivtguil3QL5a3bDE7g";
```

Comment:
It should be avoided to store any secrets in Git, even if it's used for testing.
Is it possible to create an Util class that will generate for you a token which can be used for the test?
Please investigate and let me know